src/bci_build/package.py

```diff
@@ -854,13 +854,17 @@ def description(self) -> str:
         :py:attr:`BaseContainerImage.pretty_name` to generate a description.
 
         """
+
+        description_formatters = {
```

Comment:
Please mention this in the docstring of the `description`. Also, this will certainly mess with `f""` strings, so I'm not overly happy with this change :-/

Reply:
we currently do not have f strings with custom description and I don't predict we'll likely get one soon. so I think its okay (and we can always expand those substitutions, or nest them. that also works).